EIPS/eip-draft_fct_oracle_amm.md

```diff
@@ -0,0 +1,207 @@
+---
```

Comment:
```suggestion
---
eip: 7527
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename and ETH magicians discussion

Reply:
The EIP7797 is very cool. This will be a disruptive asset issuance protocol. I think there are at least two things that are subversive:
1. Asset issuance rights are controlled by users, not the so-called project parties
2. Asset issuance has sufficient liquidity.

The project party only formulates the rules for asset issuance. Both project parties and users need to mint assets according to the rules, and these assets will automatically enter the Pool. Any asset holder can redeem their assets at any time. Not only NFT artists, but also games, social media, and even any influential organization or individual can use this standard to issue assets without worrying about unfair asset issuance and insufficient asset liquidity. Very good, looking forward to further development.

Reply:
Updated the file name and on magicians discussion as EIP-7527